packages/tools/src/tools/annotation/LivewireContourTool.ts

```diff
@@ -72,6 +70,17 @@ class LivewireContourTool extends ContourSegmentationBaseTool {
       supportedInteractionTypes: ['Mouse', 'Touch'],
       configuration: {
         preventHandleOutsideImage: false,
+        interpolation: { enabled: false, smoothHandles: 0, nearestEdge: 0 },
```

Comment:
Do we want interpolation available for the Livewire Contour Tool? I remember us deciding to only use it for segmentation tools like Spline Segmentation, Freehand Segmentation, and Livewire Segmentation.

Reply:
Smooth handles and nearesetEdge requires documentation